classes/imapemailsource.class.inc.php

```diff
@@ -79,7 +79,20 @@ public function GetMessagesCount()
 	 */
 	public function GetMessage($index)
 	{
-		$sRawHeaders = imap_fetchheader($this->rImapConn, 1+$index);
+		$sRawHeaders = imap_fetchheader($this->rImapConn, 1+$index);        
```

Comment:
Please remove the extra whitespace at the end.
```suggestion
		$sRawHeaders = imap_fetchheader($this->rImapConn, 1+$index);
```